src/agni/client/communication/MessageSender.java

```diff
@@ -1,12 +1,17 @@
 package agni.client.communication;
 
 public class MessageSender {
-	
-	String ip;
-	int port;
-	
-	public MessageSender(){
-		
-	}
+
+    private String ip;
+    private int tcpSocket;
+    private int port;
```

Comment:
`MessageSender` shouldn't need ot keep the `ip` and `port` as private members. We will add a connect method later, added it as Issue #30